cumulus_etl/etl/pipeline.py

```diff
@@ -192,7 +196,7 @@ async def run_pipeline(
             await task.init_check()
 
     # Grab a list of all required resource types for the tasks we are running
-    required_resources = set(t.resource for t in selected_tasks)
+    required_resources = set().union(*(t.get_resource_types() for t in selected_tasks))
```

Comment:
ok, i hadn't seen this pattern before, so i had to look it up. can you change the comment above to 'Grab a list of all unique required...'?

Reply:
I was frustrated that I couldn't find a simpler way to "dynamically combine some sets" - feels like something that should have a util method somewhere, I just couldn't find it.

I can add a comment, yeah.